src/main/java/io/confluent/examples/streams/interactivequeries/WordCountInteractiveQueriesExample.java

```diff
@@ -202,18 +212,22 @@ static KafkaStreams createStreams(final Properties streamsConfiguration) {
     final Serde<String> stringSerde = Serdes.String();
     StreamsBuilder builder = new StreamsBuilder();
     KStream<String, String>
-        textLines = builder.stream(stringSerde, stringSerde, TEXT_LINES_TOPIC);
+        textLines = builder.stream(TEXT_LINES_TOPIC, Consumed.with(Serdes.String(), Serdes.String()));
```

Comment:
As we set default Serdes above, we can omit `Consumed` here.

I guess, we should double check all examples if there is code that can be simplified with this regard.

Reply:
Note: This is a double-edged sword.

On the one hand, removing the serdes here is good because 1. they are not needed and 2. the code looks cleaner.

On the other hand, the examples are also a teaching tool. I have run into quite a few users who assumed that they did not have to specify serdes *at all* because they assumed (looking at the "simplified" examples) Kafka Streams would automagically detect and set the correct serdes.

Reply:
Sure, but than we should remove the serdes from `StreamsConfig`